main.go

```diff
@@ -249,7 +249,7 @@ func createCluster(cmd *cobra.Command, args []string) error {
 		fmt.Sprintf("VIRTINK_MACHINE_ROOTFS_SIZE=%s", createClusterOpts.virtinkMachineRootfsSize.String()),
 	)
 
-	generateClusterArgs := []string{"generate", "cluster", createClusterOpts.clusterName}
+	generateClusterArgs := []string{"generate", "cluster", createClusterOpts.clusterName, "--flavor", "mhc"}
```

Comment:
`--flavor` conflicts with `--from`

Reply:
`--from` allows users to customize template, we'd better keep it.  add `--flavor=mhc` only when `--from` is absent